packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -130,13 +130,9 @@ exports.sourceNodes = async (
   const nextSyncToken = currentSyncData.nextSyncToken
 
   // Store our sync state for the next sync.
-  // TODO: we do not store the token if we are using preview, since only initial sync is possible there
-  // This might change though
-  if (options.host !== `preview.contentful.com`) {
-    const newState = {}
-    newState[`${options.spaceId}-${options.environment}`] = nextSyncToken
-    setPluginStatus(newState)
-  }
+  const newState = {}
+  newState[`${options.spaceId}-${options.environment}`] = nextSyncToken
```

Comment:
I think we should have different keys for preview and delivery sync tokens

Reply:
Only if you're doing preview & final builds on the same server. Generally you would only use preview as part of a preview service e.g. Gatsby Preview ;-)

Also what you point out isn't a new problem. It might be something we consider fixing in the future but for now I think it's fine.